pallets/gear/src/schedule.rs

```diff
@@ -605,69 +647,59 @@ impl Default for Limits {
-            memory_current: cost!(instr_memory_current, 0),
-            i64clz: cost!(instr_i64clz, 1),
-            i64ctz: cost!(instr_i64ctz, 1),
-            i64popcnt: cost!(instr_i64popcnt, 1),
-            i64eqz: cost!(instr_i64eqz, 1),
-            i64extendsi32: cost!(instr_i64extendsi32, 0),
-            i64extendui32: cost!(instr_i64extendui32, 0),
-            i32wrapi64: cost!(instr_i32wrapi64, 1),
-            i64eq: cost!(instr_i64eq, 2),
-            i64ne: cost!(instr_i64ne, 2),
-            i64lts: cost!(instr_i64lts, 2),
-            i64ltu: cost!(instr_i64ltu, 2),
-            i64gts: cost!(instr_i64gts, 2),
-            i64gtu: cost!(instr_i64gtu, 2),
-            i64les: cost!(instr_i64les, 2),
-            i64leu: cost!(instr_i64leu, 2),
-            i64ges: cost!(instr_i64ges, 2),
-            i64geu: cost!(instr_i64geu, 2),
-            i64add: cost!(instr_i64add, 2),
-            i64sub: cost!(instr_i64sub, 2),
-            i64mul: cost!(instr_i64mul, 2),
-            i64divs: cost!(instr_i64divs, 2),
-            i64divu: cost!(instr_i64divu, 2),
-            i64rems: cost!(instr_i64rems, 2),
-            i64remu: cost!(instr_i64remu, 2),
-            i64and: cost!(instr_i64and, 2),
-            i64or: cost!(instr_i64or, 2),
-            i64xor: cost!(instr_i64xor, 2),
-            i64shl: cost!(instr_i64shl, 2),
-            i64shrs: cost!(instr_i64shrs, 2),
-            i64shru: cost!(instr_i64shru, 2),
-            i64rotl: cost!(instr_i64rotl, 2),
-            i64rotr: cost!(instr_i64rotr, 2),
+            version: 6,
+            i64const: cost_instr!(instr_i64const, 1),
+            i64load: cost_instr!(instr_i64load, 0),
+            i64store: cost_instr!(instr_i64store, 1),
+            select: cost_instr!(instr_select, 2),
+            r#if: cost_instr!(instr_if, 0),
+            br: cost_instr!(instr_br, 0),
+            br_if: cost_instr!(instr_br_if, 1),
+            br_table: cost_instr!(instr_br_table, 0),
+            br_table_per_entry: cost_instr!(instr_br_table_per_entry, 0),
+            call: cost_instr!(instr_call, 2),
+            call_indirect: cost_instr!(instr_call_indirect, 1),
+            call_indirect_per_param: cost_instr!(instr_call_indirect_per_param, 1),
+            call_per_local: cost_instr!(instr_call_per_local, 1),
+            local_get: cost_instr!(instr_local_get, 0),
+            local_set: cost_instr!(instr_local_set, 1),
+            local_tee: cost_instr!(instr_local_tee, 1),
+            global_get: cost_instr!(instr_global_get, 0),
+            global_set: cost_instr!(instr_global_set, 1),
+            memory_current: cost_instr!(instr_memory_current, 1),
+            i64clz: cost_instr!(instr_i64clz, 1),
+            i64ctz: cost_instr!(instr_i64ctz, 1),
+            i64popcnt: cost_instr!(instr_i64popcnt, 1),
+            i64eqz: cost_instr!(instr_i64eqz, 1),
+            i64extendsi32: cost_instr!(instr_i64extendsi32, 1),
```

Comment:
for i64extendsi32 and i64extendui32 number must be 0 because it uses only `drop` for result and `i32const` for argument. Both will be optimised and removed (i32const often when const argument can be inserted in instruction logic)